client/src/main/python/slipstream/NodeDecorator.py

```diff
@@ -70,6 +70,7 @@ class NodeDecorator(object):
     RUN_BUILD_RECIPES_KEY = 'run-build-recipes'
     PLATFORM_KEY = 'platform'
     LOGIN_USER_KEY = 'loginUser'
+    LOGIN_PASS_KEY = 'login.password'
```

Comment:
Do you want to update `loginUser` param name (server side and here) to follow the same `dot` based separator?